src/coreclr/jit/emit.cpp

```diff
@@ -3904,6 +3904,111 @@ void emitter::emitRecomputeIGoffsets()
 #endif
 }
 
+//----------------------------------------------------------------------------------------
+// emitDispCommentForHandle:
+//    Displays a comment for a handle, e.g. displays a raw string for GTF_ICON_STR_HDL
+//    or a class name for GTF_ICON_CLASS_HDL
+//
+// Arguments:
+//    handle - a constant value to display a comment for
+//    flags  - a flag that the describes the handle
+//
+void emitter::emitDispCommentForHandle(size_t handle, GenTreeFlags flag)
+{
+#ifdef DEBUG
+    if (handle == 0)
+    {
+        return;
+    }
+
+#ifdef TARGET_XARCH
+    const char* commentPrefix = "      ;";
+#else
+    const char* commentPrefix = "      //";
```

Comment:
i don't remember why we have `//` for armarch. May be just have `;` for al platforms?

Reply:
If we decide to go with `;`, we will have to change https://github.com/dotnet/runtime/pull/55970/files#diff-2b2c8b9011607926410624d6f81613fad7b74c6e0516d578675a8b792998fe4fR12431 as well.

Reply:
@kunalspathak it looks like comments in arm are // and /**/ so it's correct
https://developer.arm.com/documentation/100067/0611/armclang-Integrated-Assembler/Syntax-of-assembly-files-for-integrated-assembler